docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -135,7 +135,7 @@ Use these statements to register users, create roles, and grant and revoke privi
 | <<alter_user_statement,ALTER USER Statement>>                                 | Changes attributes associated with a user who is registered in the database.
 | <<create_role_statement,CREATE ROLE Statement>>                               | Creates an SQL role.
 | <<drop_role_statement,DROP ROLE Statement>>                                   | Deletes an SQL role.
-| <<grant_statement,GRANT Statement>>                                           | Grants access privileges on an SQL object to specified users or roles.
+| <<grant_statement,GRANT Statement>>                                           | Grants access privileges on an SQL object or an SQL objects' columns to specified users or roles.

```

Comment:
Grammar nit: should be "object's" (since we are talking about the columns of one object, not several).